### DISCUSSION TITLE
hg19 and hg38 where do the repeat Annotations come from?

### DISCUSSION THREAD

**Discussion by jo-mc** [GMOD/jbrowse-components#1691] [Q&A] [answered]
About Info on the track points to https://jbrowse.org/genomes/hg38/repeats.bb

What is the source of this info? 
Do you generate the bb file, and if so where is the code for doing this? 

Thanks.

**Comment by cmdcolin** [comment, marked as answer]
Hi @jo-mc 
I actually don't have good provenence on this file, but I think I made it awhile ago.

 It was probably generated from UCSC Table Browser, selecting repeats, RepeatMasker, get output as BED
https://genome.ucsc.edu/cgi-bin/hgTables?hgsid=1028457289_xuPfKZcYbYCm3RHzKfngPzDa9lLe&clade=mammal&org=Human&db=hg38&hgta_group=rep&hgta_track=dbSnp153Composite&hgta_table=0&hgta_regionType=genome&position=chrX%3A15%2C560%2C138-15%2C602%2C945&hgta_outputType=bed&hgta_outFileName=

This can then be converted to bigbed with bedToBigBed

I could probably create an updated version of this with a bit more provenance in the "About track" panel

**Reply by cmdcolin** [reply]
would probably be worth adding the other types of repeat tracks too !
